vispy/plot/plotwidget.py

```diff
@@ -222,7 +222,7 @@ def image(self, data, cmap='cubehelix', clim='auto', fg_color=None):
 
     def mesh(self, vertices=None, faces=None, vertex_colors=None,
              face_colors=None, color=(0.5, 0.5, 1.), fname=None,
-             meshdata=None):
+             meshdata=None, shading=None):
```

Comment:
Or did you mean I should add the appropriate code to handle the 'auto' keyword as you specify?